agenta-web/src/components/Playground/Views/ParametersCards.tsx

```diff
@@ -136,6 +141,14 @@ export const ModelParameters: React.FC<ModelParametersProps> = ({
                                                 ))}
                                             </Select>
                                         )}
+                                        {param.type === "boolean" && (
+                                            <Switch
+                                                defaultValue={param.default}
```

Comment:
`param.default` is going to be `true | false` or `1 | 0`?

Reply:
false